doc/releases/mimic.rst

```diff
@@ -1,3 +1,161 @@
+v13.2.5 Mimic
+=============
+
+This is the fifth bugfix release of the Mimic v13.2.x long term
+stable release series. We recommend all Mimic users upgrade.
+
+Notable Changes
+---------------
+
+* This release fixes the pg log hard limit bug that was introduced in
+  13.2.2, https://tracker.ceph.com/issues/36686. A flag called
+  `pglog_hardlimit` has been introduced, which is off by default. Enabling
+  this flag will limit the length of the pg log. In order to enable
+  that, the flag must be set by running `ceph osd set pglog_hardlimit`
+  after completely upgrading to 13.2.2. Once the cluster has this flag
+  set, the length of the pg log will be capped by a hard limit. Once set,
+  this flag *must not* be unset anymore. In luminous, this feature was
+  introduced in 12.2.11. Users who are running 12.2.11, and want to
+  continue to use this feauture, should upgrade to 13.2.5 or later.
```

Comment:
`s/feauture/feature/`

Reply:
"Users who are running 12.2.11, and want to continue to use this feauture, should upgrade to 13.2.5 or later."

Hm. This sentence is addressed to luminous users. I wonder whether it would make sense to include this material in the luminous 12.2.11 release notes as well?

For example, append the following after the last sentence of first bullet point of the 12.2.11 release notes: "Once set, this flag must not be unset anymore. When upgrading a cluster which has this flag set, keep in mind that Mimic received this feature only in 13.2.5, so such a cluster must be upgraded to version 13.2.5 or newer."

Reply:
I guess that makes sense to add in the 12.2.11 notes, I'll create a pr for that. I'll merge this for now since the blog is already published